client/src/components/Workflow/WorkflowDropdown.vue

```diff
@@ -9,6 +9,12 @@
             <font-awesome-icon icon="caret-down" />
             <span>{{ workflow.name }}</span>
         </b-link>
+        <font-awesome-icon
+            v-if="workflow.source_metadata && workflow.source_metadata.trs_tool_id"
+            v-b-tooltip.hover
+            :title="'Imported from TRS ID (version ' + workflow.source_metadata.trs_version_id + ')'"
+            icon="check"
+            style="color: green" />
```

Comment:
We could also do this for workflows imported from a URL, providing a link, but I'm not sure if they are worthy of the green tick :laughing: